subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/DependencyManagementProjectScopeServices.java

```diff
@@ -0,0 +1,97 @@
+import org.gradle.api.internal.artifacts.ivyservice.ArtifactCacheMetadata;
+import org.gradle.api.internal.artifacts.transform.TransformationNodeDependencyResolver;
+import org.gradle.api.internal.attributes.ImmutableAttributesFactory;
+import org.gradle.api.internal.file.FileCollectionFactory;
+import org.gradle.api.internal.filestore.DefaultArtifactIdentifierFileStore;
+import org.gradle.api.internal.notations.ClientModuleNotationParserFactory;
+import org.gradle.api.internal.notations.DependencyConstraintNotationParser;
+import org.gradle.api.internal.notations.DependencyNotationParser;
+import org.gradle.api.internal.notations.ProjectDependencyFactory;
+import org.gradle.api.internal.runtimeshaded.RuntimeShadedJarFactory;
+import org.gradle.api.internal.tasks.TaskDependencyFactory;
+import org.gradle.internal.installation.CurrentGradleInstallation;
+import org.gradle.internal.reflect.Instantiator;
+import org.gradle.internal.resource.cached.ByUrlCachedExternalResourceIndex;
+import org.gradle.internal.resource.cached.DefaultExternalResourceFileStore;
+import org.gradle.internal.resource.cached.ExternalResourceFileStore;
+import org.gradle.internal.service.ServiceRegistration;
+import org.gradle.internal.typeconversion.NotationParser;
+import org.gradle.util.internal.BuildCommencedTimeProvider;
+import org.gradle.util.internal.SimpleMapInterner;
+
+/**
+ * The set of dependency management services that are created per project.
+ */
+class DependencyManagementProjectScopeServices {
+    void configure(ServiceRegistration registration) {
+        registration.add(DefaultExternalResourceFileStore.Factory.class);
+        registration.add(DefaultArtifactIdentifierFileStore.Factory.class);
+        registration.add(TransformationNodeDependencyResolver.class);
+    }
+
+    /** This is needed in order to get the {@code taskDependencyFactory} from the current project. */
+    DefaultProjectDependencyFactory createProjectDependencyFactory(
+        Instantiator instantiator,
+        StartParameter startParameter,
+        ImmutableAttributesFactory attributesFactory,
+        TaskDependencyFactory taskDependencyFactory
+    ) {
+        NotationParser<Object, Capability> capabilityNotationParser = new CapabilityNotationParserFactory(false).create();
+        return new DefaultProjectDependencyFactory(instantiator, startParameter.isBuildProjectDependencies(), capabilityNotationParser, attributesFactory, taskDependencyFactory);
+    }
+
+    DependencyFactoryInternal createDependencyFactory(
+        Instantiator instantiator,
+        DefaultProjectDependencyFactory factory,
+        ClassPathRegistry classPathRegistry,
+        CurrentGradleInstallation currentGradleInstallation,
+        FileCollectionFactory fileCollectionFactory,
+        RuntimeShadedJarFactory runtimeShadedJarFactory,
+        ImmutableAttributesFactory attributesFactory,
+        SimpleMapInterner stringInterner) {
```

Comment:
💄 I'd expect the closing parenthesis and opening brace to be on the next line (just like the method above).